### ISSUE TITLE
setup_env.py offline mode

### ISSUE THREAD

**Issue by elliottslaughter** [StanfordLegion/legion#347]
To allow setup_env.py to be used in a computer without a direct internet connection.

**Comment by elliottslaughter** [comment]
Initial support landed in 262d67dbc4a16fd3748fbe08f2752aa1aaae186e. To test:

```bash
# from machine with internet connection
./scripts/setup_env.py --prefix ./install --cache

# copy the ./install directory to remote machine

# from remote machine
CC=gcc CXX=g++ ./scripts/setup_env.py --prefix ./install
```

**Comment by elliottslaughter** [comment]
This has been in for some time. Specific problems should be reported as new issues.
